autoPyTorch/pipeline/components/base_component.py

```diff
@@ -127,14 +127,11 @@ def get_fit_requirements(self) -> Optional[List[FitRequirement]]:
         return self._fit_requirements
 
     def add_fit_requirements(self, requirements: List[FitRequirement]) -> None:
-        if self._fit_requirements is not None:
-            self._fit_requirements.extend(requirements)
-        else:
-            self._fit_requirements = requirements
+        self._fit_requirements.extend(requirements)
 
     @staticmethod
     def get_properties(dataset_properties: Optional[Dict[str, str]] = None
-                       ) -> Dict[str, Any]:
+                       ) -> Dict[str, Union[str, bool]]:
```

Comment:
Could you give an example what could be the bool value for this dict in the doc string?

Reply:
see [this](https://github.com/automl/Auto-PyTorch/blob/1818445d25415d9396ccd10fd7057dc315666418/autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/encoding/OneHotEncoder.py#L35)